cs41_final.sh

```diff
@@ -10,12 +17,19 @@ show_banner(){
   echo -e "${MENU}*********************************************${NORMAL}"
   echo -e "${MENU}*************** System Report ***************${NORMAL}"
   echo -e "${MENU}*********************************************${NORMAL}"
-  echo -e "${MENU}${NUMBER} Name of Server:${MENU} $servName ${NORMAL}"
-  echo -e "${MENU}${NUMBER} Server IP Address:${MENU} $servIP ${NORMAL}"
-  echo -e "${MENU}${NUMBER} Number of Current Users: ${MENU} $numUsers ${NORMAL}"
-  echo -e "${MENU}${NUMBER} Date and Time:${MENU} $dateTime ${NORMAL}"
-  echo -e "${MENU}${NUMBER} OS Version:${MENU} $osVersion ${NORMAL}"
-  echo -e "${MENU}${NUMBER} Current User Name:${MENU} $userName ${NORMAL}"
+  echo -e " ";
+  echo -e "${MENU}${NUMBER}Name of Server:${MENU}\n$(hostname -s)${NORMAL}"
+  echo -e "${MENU}    ****-****-****-****  ${NORMAL}"
+  echo -e "${MENU}${NUMBER}Server IP Address:${MENU}\n$(hostname -i)${NORMAL}"
+  echo -e "${MENU}    ****-****-****-****  ${NORMAL}"
+  echo -e "${MENU}${NUMBER}Number of Current Users:${MENU}\n$(who)${NORMAL}"
+  echo -e "${MENU}    ****-****-****-****  ${NORMAL}"
+  echo -e "${MENU}${NUMBER}Date and Time:${MENU}\n$(date)${NORMAL}"
+  echo -e "${MENU}    ****-****-****-****  ${NORMAL}"
+  echo -e "${MENU}${NUMBER}OS Version:${MENU}\n$(lsb_release -a)${NORMAL}"
```

Comment:
this is not portable.  for instance, on my ubuntu machine, lsb is not installed.  
The safer, easier route would be to run:
uname -a

which just lists details about the kernel installed, and should be portable.